doc/field_guide__flaring.md

```diff
@@ -0,0 +1,28 @@
+## Flaring variables (flaring)
+
+The phenomelogical classification of ``flaring`` is any kind of outbursting source, where a variable star (or binary system) dramatically increases in brightness for minutes to hours before returning to quiescence. Most flare stars are red dwarfs, while RS Canum Venaticorum variables (RS CVn) are also known to flare due to a companion star in the binary system. Other possible sources of flaring include dwarf novae which are increases in luminosity in the accretion disk of a cataclysmic variable star.

```

Comment:
Reword. Dwarf novae are not an increase in luminosty. "... increases are ..." also do not juxtapose well.